docs/concepts/storage/persistent-volumes.md

```diff
@@ -219,6 +219,13 @@ Current reclaim policies are:
 * Delete -- associated storage asset such as AWS EBS, GCE PD, Azure Disk, or OpenStack Cinder volume is deleted
 
 Currently, only NFS and HostPath support recycling. AWS EBS, GCE PD, Azure Disk, and Cinder volumes support deletion.
+In case of Retain policy, admin can make a volume as reusable by following manual reclamation steps.
+- Delete the `PersistentVolumeClaim`. `PersistentVolume` remains after deletion of PVC.
+- Delete the `PersistentVolume`. The volume on storage asset remains after deletion of PV.
+- Create a new `PersistentVolume` with the volume definition.
```

Comment:
Before the create step, "Manually clean up the data on the volume accordingly"

For the create step, "Manually delete the volume, or if you want to reuse the same volume,"